garrysmod/lua/autorun/base_npcs.lua

```diff
@@ -87,31 +90,31 @@ AddNPC( {
 	Class = "npc_citizen",
 	Category = Category,
 	Model = "models/odessa.mdl",
-	KeyValues = { citizentype = CT_UNIQUE }
+	KeyValues = { citizentype = CT_UNIQUE, SquadName = "resistance" }
 }, "npc_odessa" )
 
 AddNPC( {
 	Name = "Medic",
 	Class = "npc_citizen",
 	Category = Category,
-	SpawnFlags = SF_CITIZEN_MEDIC,
-	KeyValues = { citizentype = CT_REBEL },
+	SpawnFlags = bit.bor( 8, 131072 ),

```

Comment:
Why use raw values here and not `bit.bor( SF_NPC_DROP_HEALTHKIT, SF_CITIZEN_MEDIC )`?